dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNInjectorInterfaceIIDM.h

```diff
@@ -120,9 +120,12 @@ class InjectorInterfaceIIDM {
  protected:
   powsybl::iidm::Injection& injectorIIDM_;                        ///< reference to the iidm injector instance
   std::string injectorId_;                                        ///< injector's id
-  boost::shared_ptr<BusInterface> busInterface_;                  ///< busInterface of the bus where the injector is connected
+//  boost::shared_ptr<BusInterface> busInterface_;                  ///< busInterface of the bus where the injector is connected
```

Comment:
Please remove this useless comment

Reply:
Of course I will, di you notice I changed from protected to private ? May I do that for other protected members whenever it is possible ?

Reply:
removed